proposals/README.md

```diff
@@ -17,61 +35,46 @@ To execute the deployment run:
+* exec - When true, the e2e tests will run the actual proposal through the dao instead of the `run` hook which mocks the behavior. It imports the proposal steps from the `proposals/description/fip_x.json` file
 
-These are validated in the last e2e test
-
-## Optional Step 5: Deploying and Updating Addresses
-If your contract has a deployment step from Step 2, you need to deploy your new contracts to mainnet before moving on to Step 6.
+## Optional Step 4: Deploying and Updating Addresses
+If your contract has an optional deployment step from above, you need to deploy your new contracts to mainnet before moving on to Step 5.
 
 Run your deploy script if you had one from step 2. Update `/contract-addresses/mainnet-addresses.json` with the new contract addresses. 
 
 Update the fork block inside the hardhat config and set the deploy flag to false in the config entry for `fip_x` in `end-to-end/proposals_config.json`
 
 Finally rerun `npm run test:e2e` and make sure everything passes as expected.
 
-## Step 6: Testing Exact Proposal Calldata
-
-This script does a full end-to-end DAO proposal, vote, queue, and execution on the exact calldata to be submitted to the DAO.
-
-The goal of this step is to compare the execution results of the calldata to the mock script from step 4. Any setup and teardown calls in the `proposals/dao/fip_x.js` file will still execute, but the run step is replaced by the execution of the DAO code in `proposals/dao/exec.js`.
+## Step 5: Testing Exact Proposal
 
-First compose the calldata, we recommend using the Tally interface + metamask (if you want to build a tool to construct it within the repo using the json descriptions, apply for a grant!).
+This script does a full end-to-end DAO proposal, vote, queue, and execution on the exact proposal to be submitted to the DAO.
 
-Make sure to have visibility into the calldata on your browser wallet. In Metamask this is 
-in settings -> advanced -> show hex data. Upon Tx submission you will read this data.
+The goal of this step is to compare the execution results of the calldata to the mock script from step 3. Any setup and teardown calls in the `proposals/dao/fip_x.js` file will still execute, but the run step is replaced by the execution of the proposal steps in `proposals/description/fip_x.json`.
 
-To get the calldata, submit a proposal on https://www.withtally.com/governance/fei/proposal/new with the intended proposal commands, title and description. Click "Submit proposal" but **do not** confirm the Metamask transaction. 
-
-Copy the calldata into the config entry for `fip_x` in `end-to-end/proposals_config.json` for integration testing:
-* exec - set to true
-* proposal_calldata - fill in the copied calldata
-* proposer - fill in proposer address
-* voter - 0xB8f482539F2d3Ae2C9ea6076894df36D1f632775
+Set the exec flag to true into the config entry for `fip_x` in `end-to-end/proposals_config.json`
```

Comment:
This step will only be removable once we don't execute the proposal via js script, but exclusively via calldata from json file right?

Reply:
We'd need to remove the ability to include non-deployed contracts to the proposal which requires manipulating the json at runtime after doing the deploy script.